### DISCUSSION TITLE
Type lost for useWatch

### DISCUSSION THREAD

**Discussion by AnjuBudhwanBala** [react-hook-form/react-hook-form#2657] [Q&A] [answered]
**Describe the bug**
Type is missing for useWatch

**To Reproduce**
This import line : import { useForm, useWatch } from "react-hook-form"

`Module '"../../../node_modules/react-hook-form/dist"'  has no exported member 'useWatch'. TS2305`

**versions**
react-hook-form: "6.4.1",
typescript: "3.9.7"

**Comment by bluebill1049** [comment, marked as answer]
what type do you need? we only export type which are documented in the website.

**Reply by AnjuBudhwanBala** [reply]
I am really sorry for pasting an incorrect link of expected behaviour here. I am trying to import "useWatch" but I am getting this error "Module '"../../../../../node_modules/react-hook-form/dist"' has no exported member 'useWatch'." Do you know what I am missing here?

**Reply by bluebill1049** [reply]
that's strange, it sould just worked, can you share a codesandbox to reproduce the issue.

**Reply by AnjuBudhwanBala** [reply]
Here you go: https://codesandbox.io/s/react-hook-form-get-started-forked-1db62?file=/src/index.tsx

**Reply by bluebill1049** [reply]
you are using wrong version. v6 plz

**Reply by AnjuBudhwanBala** [reply]
This is fixed now. All I was using wrong version. Thank you so much :)
